tools/permuter_settings.toml

```diff
@@ -1,6 +1,7 @@
 [preserve_macros]
 "g[DS]P.*" = "void"
 "gDma.*" = "void"
+"a[A-Z].*" = "void"
```

Comment:
not familiar with the permuter, is this fine to commit?

Reply:
It should be yeah, I assume it's to preserve abi commands (the equivalent of gbi commands for audio).
And I don't think it should conflict with any other macros since it wouldn't fit our naming scheme